App.js

```diff
@@ -89,8 +89,12 @@ export default function App() {
 
   const fetchUserData = async () => {
     console.log('fetchUserData invoked');
-    await axios.get(`http://18.219.200.72:8080/user/?user_id=${userId}`)
+    axios.get(`http://18.219.200.72:8080/user/?user_id=${userId}`)
     .then(function (response) {
+      console.log('USER DATA', response.data)
+      var base64Image = response.data[0].thumbnail.toString('base64');
+      response.data[0].thumbail = base64Image
```

Comment:
This is a really interesting implementation! Is there an advantage to doing this over referencing the url?